src/hooks/root-hooks/use-storage-analytics.ts

```diff
@@ -0,0 +1,38 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import { useCallback } from 'react';
+
+import { AnalyticsEventCategory } from 'src/utils/analytics/analytics-event.enum';
+import { useAnalytics } from 'src/utils/analytics/use-analytics.hook';
+import { useTimeout } from 'src/utils/hooks';
+import { calculateStringSizeInBytes } from 'src/utils/string.utils';
+
+export const useStorageAnalytics = () => {
+  const { trackEvent } = useAnalytics();
+
+  const measureAndSend = useCallback(
+    () =>
+      getAsyncStorageUsageDetails().then(asyncStorage =>
+        trackEvent('STORAGES_STATE', AnalyticsEventCategory.General, { asyncStorage })
+      ),
+    [trackEvent]
+  );
+
+  // Delaying to not throttle initial app load
+  useTimeout(measureAndSend, 10000);
```

Comment:
why 10s? mb there is a device where initial load would take longer.. and if there is a problem with throttle mb we need the same timeout in `usePushNotifications`?